.travis.yml

```diff
@@ -23,6 +26,9 @@ install: |
   if [ $? != 0 ]; then
     curl -sSf https://sh.rustup.rs | sh -s -- -y
   fi
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    brew install ccache && export PATH="/usr/local/opt/ccache/libexec:$PATH";
```

Comment:
It doesn't have it already?

Reply:
apparently not: https://docs.travis-ci.com/user/caching/#ccache-on-os-x